packages/shared/src/components/Attachments/AttachmentItem.tsx

```diff
@@ -0,0 +1,219 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { Asset, useAssetUpload, useFileInput } from '@frontify/app-bridge';
+import { useSortable } from '@dnd-kit/sortable';
+import {
+    ActionMenu,
+    Button,
+    ButtonEmphasis,
+    Flyout,
+    FlyoutPlacement,
+    IconArrowCircleUp20,
+    IconDocument24,
+    IconGrabHandle20,
+    IconImage24,
+    IconImageStack20,
+    IconPen20,
+    IconPlayFrame24,
+    IconTrashBin20,
+    MenuItemContentSize,
+    MenuItemStyle,
+} from '@frontify/fondue';
+import { forwardRef, useEffect, useState } from 'react';
+import { AttachmentItemProps, SortableAttachmentItemProps } from './types';
+
+const getDecorator = (type: string) => {
```

Comment:
A specific decorator for audio files would be nice

Reply:
good idea, implemented